### PR TITLE
Fixed table column issue when switching webfilter.

### PR DESCRIPTION
**Pull request by nxanil**
@myTechPlayArea @preshitg 

Fixed Issue: https://github.com/nuagenetworks/vis-config/issues/247.
Please review the PR.

### CODE REVIEW

File: Graphs/Table/index.js [hunk 1]
```diff
@@ -103,6 +103,26 @@ const setSelectedRows = (props) => {
...
+const selectedContext = (props) => {
+    const { context, columns } = props;
+    let key = [];
+    columns.forEach(column => {
+        if(column.displayOption){
+            key.push(...Object.keys(column.displayOption));
+        }
+    });
+    key = uniq(key);
+    let queryKey = 'ALL';
```
Comment (preshitg) [inline Graphs/Table/index.js:117]: You have a default of ALL. And at L120 you are appending more keys without delimiter. Can you give an example here? It is difficult to understand the need for this function.

**Review by preshitg — COMMENTED** [review]
As with most of the PR's, can you add a screenshot / gif showing working scenario?

File: Graphs/Table/index.js (hunk 1 above)
Comment (nxanil) [reply Graphs/Table/index.js:149]: Done.

**Comment by preshitg** [comment]
@myTechPlayArea Could you please check this once since you have raised this issue?

**Comment by myTechPlayArea** [comment]
Looks good. Tested with ACL data as well and the default columns are loading properly. OK to merge

**Review by myTechPlayArea — APPROVED** [review]

**Pull request merged**